### PR TITLE
[#1309] - Add news editor role

### LINKED ISSUE #1309: Create "news editor" role
This new role could have the following permissions:

* News article: Create new content
* News article: Delete own content
* News article: Edit any content
* News article: Edit own content
* News article: View revisions

and would not have the following permissions:

* News article: Delete revisions
* News article: Delete any content
* News article: Revert revisions

### PR DESCRIPTION
**Pull request by Saphyel**
Fixes #1309

## How to test
* Login `news_editor` : `news_editor`
* Users with new "News editor" role can create and edit news articles including access to entities required for those (e.g. media, pdf..). 
* They should see the administration theme when editing the content
* They shouldn't be able to do anything else.

### CODE REVIEW

File: config/install/user.role.news_editor.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+status: true
+dependencies: {  }
+id: news_editor
+label: 'News editor'
+weight: -4
+is_admin: null
+permissions:
+  - 'access in-place editing'
+  - 'access media overview'
+  - 'administer redirects'
```
Comment (adamclark-dev) [inline config/install/user.role.news_editor.yml]: Do we want them to have this role?

File: config/install/image.style.bg_rich_text_narrow.yml [hunk 2]
```diff
@@ -13,3 +13,4 @@ effects:
     data:
       width: '740'
       height: '663'
+      crop_type: focal_point
```
Comment (pvhee) [inline config/install/image.style.bg_rich_text_narrow.yml]: @Saphyel why do you have all those unrelated commits in this PR - this doesn't look right?

File: config/install/user.role.news_editor.yml (hunk 1 above)
Comment (pvhee) [inline config/install/user.role.news_editor.yml]: they shouldn't be able to administer this

File: config/install/user.role.news_editor.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+langcode: en
+status: true
+dependencies: {  }
+id: news_editor
+label: 'News editor'
+weight: -4
+is_admin: null
+permissions:
+  - 'access in-place editing'
```
Comment (pvhee) [inline config/install/user.role.news_editor.yml]: they don't need access to this

File: config/install/user.role.news_editor.yml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  - 'access media overview'
+  - 'administer redirects'
+  - 'create article content'
+  - 'create media'
+  - 'create url aliases'
+  - 'delete any media'
+  - 'delete media'
+  - 'delete own article content'
+  - 'delete terms in article_category'
```
Comment (pvhee) [inline config/install/user.role.news_editor.yml]: they shouldn't be able to change terms

File: config/install/user.role.news_editor.yml [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+  - 'edit own article content'
+  - 'edit terms in article_category'
+  - 'update any media'
+  - 'update media'
+  - 'view any unpublished article content'
+  - 'view article revisions'
+  - 'view scheduled update entities'
```
Comment (pvhee) [inline config/install/user.role.news_editor.yml]: They don't seem to have access to the admin theme? How will they be able to successfully edit content?

File: config/install/user.role.news_editor.yml (hunk 3 above)
Comment (Saphyel) [reply config/install/user.role.news_editor.yml]: I think that makes them life easy...

File: config/install/user.role.news_editor.yml (hunk 5 above)
Comment (Saphyel) [reply config/install/user.role.news_editor.yml]: what do you mean?

**Comment by alexjohnston266** [comment]
@Saphyel everything looks good to me, thanks Carlos.

**Comment by pradip3271** [comment]
Tested on CR and SR env - Passed QA, ready for release!
